examples/automm/example_tabular.py

```diff
@@ -0,0 +1,36 @@
+from autogluon.text.automm import AutoMMPredictor
+import argparse
+from utils import get_tabular_data 
+
+def main(args):
+
+    train_df, val_df, test_df, column_types, problem_type, metric = get_tabular_data(
+        base_path=args.dataset_dir,
+        dataset_name=args.dataset_name,
+    )
+
+    predictor = AutoMMPredictor(
+        label='0', 
+        problem_type=problem_type,
+        eval_metric=metric,
+        path=args.exp_dir,
+        verbosity=4,
+    )
+
+    predictor.fit(
+        train_data=train_df,
+        tuning_data=test_df,
```

Comment:
Why are we passing test data as tuning data????
Why are we never using val_df?
How are you calculating scores for a dataset, that isn't part of this code?

Reply:
I think it should be `val_df` @yiqings

Reply:
Yes. Updated.